script/audit_deps.py

```diff
@@ -31,25 +35,32 @@ def audit_deps(args):
     output, error_data = audit_process.communicate()
 
     try:
+        # results from audit
         result = json.loads(str(output))
     except ValueError:
         # This can happen in the case of an NPM network error
         print('Audit failed to return valid json')
         return 1
 
-    resolutions, non_dev_exceptions = extract_resolutions(result)
+    if whitelisted_advisories:
```

Comment:
minor, but we can remove this `if` condition if whitelisted_advisories is always an array (it's just empty when therre are none)

Reply:
Updated